appium/tests/specs/mock/composeEmail/CheckAttachmentAfterForward.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { MockApi } from 'api-mocks/mock';
+import { MockApiConfig } from 'api-mocks/mock-config';
+import {
+  EmailScreen,
+  MailFolderScreen,
+  SetupKeyScreen,
+  SplashScreen
+} from '../../../screenobjects/all-screens';
+
+describe('COMPOSE EMAIL: ', () => {
+
+  it('check attachment after forward', async () => {
+    const mockApi = new MockApi();
+    const subject = 'Test attachment after forward';
```

Comment:
Let's use existing mock email with subject `email with text attachment`, as it has almost the same content, so we won't have several mock emails with similar content.

Reply:
Sounds good